src/main/java/core/basesyntax/service/impl/ReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Reader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderImpl implements Reader {
+
+    @Override
+    public List<String[]> read(String filePath) {
+        List<String[]> linesArray = new ArrayList<>();
+        String line;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(filePath))) {
+            bufferedReader.readLine();
```

Comment:
Skipping the first line of the file assumes that it will always be a header or some non-data line. If that's the case, it should be documented, or better yet, make it configurable whether to skip the first line or not.